toolkit_tmb/1.0.0/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rocker/tidyverse:4.2.1
+LABEL maintainer="Alex Sickler <sicklera@chop.edu>"
+LABEL description="tidyverse image with tmb 1.0.0 package"
+
+ENV OPTPARSE_VERSION 1.7.3
+ENV BIOC_VERSION 3.16
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
What is the purpose of this pipefail?

Reply:
Yeah it doesn't appear to be doing anything here. Best to get rid of it.

Reply:
Done.